### PR TITLE
Civ 2521 check your answer when pay

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-2421

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/common/form/models/admission/fullAdmission/paymentOption/paymentOption.ts [hunk 1]
```diff
@@ -15,4 +15,8 @@ export default class PaymentOption extends Form {
   paymentOptionBySetDateSelected() {
     return this.paymentType && this.paymentType === PaymentOptionType.BY_SET_DATE;
   }
+
```
Comment (TatsyMaslova) [inline src/main/common/form/models/admission/fullAdmission/paymentOption/paymentOption.ts:18]: I will remove this once the story from Maro goes into master

File: src/main/routes/features/response/admission/fullAdmission/paymentOption/paymentOptionController.ts [hunk 2]
```diff
@@ -20,6 +25,8 @@ function renderView(form: PaymentOption, res: express.Response) {
 function redirectToNextPage(claimId: string, form: PaymentOption, res: express.Response) {
   if (form.paymentOptionBySetDateSelected()) {
     res.redirect(constructResponseUrlWithIdParams(claimId, CITIZEN_PAYMENT_DATE_URL));
+  } else if (form.paymentOptionByInstallmentsSelected()) {
```
Comment (TatsyMaslova) [inline src/main/routes/features/response/admission/fullAdmission/paymentOption/paymentOptionController.ts:28]: I will remove this once the story from Maro goes into master

File: src/main/services/features/response/checkAnswersService.ts [hunk 3]
```diff
@@ -33,6 +36,16 @@ const getDefendantFullName = (claim: Claim): string => {
   return claim.respondent1.partyName;
 };
 
+const getPaymentOption = (claim: Claim, lang: string | unknown): string => {
+  const option = t(`COMMON.PAYMENT_OPTION.${claim.paymentOption}`, {lng: getLng(lang)});
+  switch (claim.paymentOption) {
```
Comment (laarmada) [inline src/main/services/features/response/checkAnswersService.ts:41]: only is my personal opinion, for two options not make sense to implement switch

File: src/main/services/features/response/checkAnswersService.ts [hunk 4]
```diff
@@ -42,6 +55,11 @@ const buildSummarySections = (claim: Claim, claimId: string, lang: string | unkn
   };
 };
 
+const buildExplanationRow = (claim: Claim, claimId: string, lang: string | unknown): SummaryRow => {
+  const explanationHref = CITIZEN_EXPLANATION_URL.replace(':id', claimId);
```
Comment (GorilaNaranja) [inline src/main/services/features/response/checkAnswersService.ts:59]: Not sure if you can use it here but we have "constructResponseUrlWithIdParams" functions that already replace id from a url, maybe you can use it

File: src/main/services/features/response/checkAnswersService.ts (hunk 3 above)
Comment (TatsyMaslova) [reply src/main/services/features/response/checkAnswersService.ts]: thank you Sergio. yes i prefer as you do. forgot to refactor

**Review by laperla11 — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request closed without merging**